.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,46 @@
+name: Pull requests
+
+on:
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+
+  build_job:
+    name: Build Pull Request
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Build Project
+        run: bash contrib/build.sh
+
+      - name: Compile Project
+        run: |
```

Comment:
For new folder structure we don't need to go into android-libkiwixbuild we should remove this. now we can directly use ```./gradlew androidSourcesJar build```